Makefile

```diff
@@ -22,8 +22,8 @@ help:
 test:
 	$(TEST_COMMAND)
 pep8:
-	flake8 micronota setup.py
+	flake8 --exit-zero micronota setup.py
```

Comment:
it prints warnings but will exit with zero. I argue pep8 is guideline not the law. I try to follow, but sometimes it just make nonsense to break unbreakable line > 79 chars.